### PR TITLE
Update for terraform 0.12

### PR DESCRIPTION
**Pull request by alex-goncharov**
This is an attempt to update this module to work with terraform 0.12. It has been tested and is currently used to manage our infra.

### CODE REVIEW

File: examples/complete-vpc/outputs.tf [hunk 1]
```diff
@@ -1,60 +1,60 @@
...
   description = "The ID of the VPC"
-  value       = "${module.vpc.vpc_id}"
+  value       = module.vpc.vpc_id
 }
 
 # Subnets
 output "private_subnets" {
   description = "List of IDs of private subnets"
-  value       = ["${module.vpc.private_subnets}"]
+  value       = [module.vpc.private_subnets]
```
Comment (Skaronator) [inline examples/complete-vpc/outputs.tf]: Not sure if this really need to be a array/list. I think passing the variable directly should be fine but not sure.

File: versions.tf [hunk 2]
```diff
@@ -0,0 +1,4 @@
+
+terraform {
+  required_version = ">= 0.12"
```
Comment (Skaronator) [inline versions.tf]: You need to increase the version in the main.tf file here:
https://github.com/alex-goncharov/terraform-aws-vpc/blob/2626e910146e7c0afee8ee8338ea3d6e048059dc/main.tf#L1-L3
and remove the versions.tf file

Comment (alex-goncharov) [reply versions.tf]: Thanks, fixed.

File: examples/complete-vpc/outputs.tf (hunk 1 above)
Comment (alex-goncharov) [reply examples/complete-vpc/outputs.tf]: Indeed. Corrected the rest of the examples as well.

**Comment by antonbabenko** [comment]
@alex-goncharov Thanks for this PR! I will review&merge few others PRs during an hour or two and then make a big-bang update for 0.12 with a deeper look into this PR and PR #228.

**Comment by antonbabenko** [comment]
[v2.0.0](https://github.com/terraform-aws-modules/terraform-aws-vpc/releases/v2.0.0) has been released.

Thanks to everyone involved, and please report if you find any issues.

**Pull request closed without merging**